repositories.bzl

```diff
@@ -29,7 +29,7 @@ VERSIONS = {
         "org": "grpc",
         "repo": "grpc",
         "ref": "v{}".format(GRPC_VERSION),
-        "sha256": "b2f2620c762427bfeeef96a68c1924319f384e877bc0e084487601e4cc6e434c",
+        "sha256": "271bdc890bf329a8de5b65819f0f9590a5381402429bca37625b63546ed19e54",
```

Comment:
just curious, how did you obtain those sha256? Is there a standard way

Reply:
I went through a pretty manual way to compute the sha256. I am wrapping up a script to automate it.